Common Programming Concepts/Functions/Extract Method Refactoring/task.md

```diff
@@ -0,0 +1,37 @@
+## Mastering the IDE: Extract Method Refactoring
+ 
+A developer was asked to change the message format in the code you see in this task.
+
+Instead of "1993: 10 years ago was 1983", we want to print "1993: ten years ago was 1983".
+
+However, the way it's implemented now, the developer has to modify the message in two places, which would 
+become even more inconvenient when this code is used in more places.
+
+So before actually changing the message format, the developer decided to perform refactoring and create a function that prints the message.
+
+###Task
+
+**Step 1: Create Function**
+
+Perform the refactoring and create a new function to replace the duplicated code.
+
+Select the first occurrence of 
+
+```rust
+println!("{}: 10 years ago was {}", year1993, year1993 - 10);
+```
+
+and then either press &shortcut:ExtractMethod; or choose *Refactor&rarr;Extract Method...* from the right-click menu.
```

Comment:
Unfortunately, the \&rarr; is not rendered correctly on WIn10/ 2021.5-2021.1-637 plugin, it is rendered as a "?"
Maybe we can switch to -> for the time being? I'll create an issue about rendering in the YT